django/db/models/sql/compiler.py

```diff
@@ -439,6 +439,9 @@ def as_sql(self, with_limits=True, with_col_aliases=False):
         try:
             extra_select, order_by, group_by = self.pre_sql_setup()
             for_update_part = None
+            # Indicates whether we need to construct the SQL needed to use
```

Comment:
Maybe: `# Is a LIMIT/OFFSET clause needed?`

(Avoid use of "we" in comments per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).)